src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs

```diff
@@ -147,7 +158,14 @@ public async Task<IActionResult> StartOrResumeApplication()
         {
             return await StartApplication();
         }
-        
+
+        public IActionResult FinancialExpiredRedir()
```

Comment:
I don't think this FinancialExpiredRedir is even used?

Reply:
Resolving this, after digging I can see it on the form, I was focusing on code behind.